batcher/aligned-batcher/src/lib.rs

```diff
@@ -806,8 +806,8 @@ impl Batcher {
 
         // FIXME(marian): This condition should be changed to current_batch_size == 0
         // once the bug in Lambdaworks merkle tree is fixed.
-        if current_batch_len < 2 {
-            info!("Current batch is empty or length 1. Waiting for more proofs...");
+        if current_batch_len == 0 {
+            info!("Current batch is empty. Waiting for more proofs...");
             return None;
         }
```

Comment:
The FIXME should be removed if it solved. Also, are we using the `current_batch_len` variable anywhere else? If not, then we can remove it and change it to
```rust
if batch_state.batch_queue.is_empty()
```

Reply:
Comment removed and length check changed